### PR TITLE
Optimistic load

### PR DESCRIPTION
**Pull request by ana2201**
## Description
Don't show so many spinners, just show the 'posting' step and the success notification as confirmation

## Preview
---> https://forum-staging-lirlbyg9g-usedispatch.vercel.app/

### CODE REVIEW

**Comment by viksit** [comment]
this looks good function wise.

- "Replace "posting" status in the post with "Confirming on chain"
- lets put that transaction ID we used to have in that same area with a chain ⛓ icon.
- once the post is done, replace "confirming on chain" with "confirmed"

**Comment by ana2201** [comment]
@viksit I added a small spinner in the buttons when awaiting for the user approval

- https://www.loom.com/share/abcd2865ef004e7f9813f73b160b1419
- https://www.loom.com/share/fd5151a73684426a9ec31272c7b05eb1

File: src/components/forums/topic/TopicContent.tsx
Comment (zfaizal2) [inline src/components/forums/topic/TopicContent.tsx:197]: same as above

**Review by zfaizal2 — CHANGES_REQUESTED** [review]

**Comment by zfaizal2** [comment]
noticed a bug where when creating a reply, two replies show up in the UI. 
image

**Comment by ana2201** [comment]
> noticed a bug where when creating a reply, two replies show up in the UI. image

oh, good catch! the order in which the replies and text area are shown were changed after this PR was opened, so when that was merged, the block ended up duplicated here 🙃

It's working fine now
image

**Pull request merged**
